variables.js

```diff
@@ -0,0 +1,17 @@
+function variables() {
+
+    let userinput;
+    userinput = document.getElementById("user").value;
```

Comment:
Two things:

a) It's unnecessary to declare the variable on a separate line from where you set it's value.

b) `let` is the wrong keyword in this case. Javascript has three kinds of variable declarations: `let`, `const`, and `var`. You should do further research into all three kinds and fix this declaration.

Reply:
This went unaddressed?